config/application.rb

```diff
@@ -66,7 +66,8 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     config.i18n.default_locale = ENV["LOCALE"]
-
+    I18n.locale = config.i18n.locale = config.i18n.default_locale
```

Comment:
The emails sent by delayed_job were not translated. See discussion  with @RohanM here : http://community.openfoodnetwork.org/t/localisation-for-delayed-job/465